packages/notebook/src/actions.tsx

```diff
@@ -1966,6 +2000,19 @@ namespace Private {
     activeCell: Cell | null;
   }
 
+  export function isNotebookRendered(notebook: Notebook): boolean {
+    if (notebook.cellsToRender.size !== 0) {
+      showDialog({
+        body: `Notebook is still rendering and has for now ${notebook.cellsToRender.size} remaining cells to render.
+
+Please wait for the complete rendering before invoking that action.`,
```

Comment:
This needs to be translated.

Reply:
I also faced that question while implementing. The translator is not given in the method signature of the NotebookActions class, and it looks overkill to me to pass that in all the static methods. Any idea/advice on that?

Reply:
I have added the default translator in the isNotebookRendered